core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -76,24 +76,32 @@ object TypelevelPlugin extends AutoPlugin {
         java <- githubWorkflowJavaVersions.value.tail // default java is head
       } yield MatrixExclude(Map("scala" -> scala, "java" -> java.render))
     },
-    GlobalScope / tlCommandAliases ++= {
+    GlobalScope / tlPrePRSteps ++= {
       val header = tlCiHeaderCheck.value
       val scalafmt = tlCiScalafmtCheck.value
       val javafmt = tlCiJavafmtCheck.value
       val scalafix = tlCiScalafixCheck.value
+      val mima = tlCiMimaBinaryIssueCheck.value
 
-      val prePR = List("project /", "githubWorkflowGenerate") ++
+      List("githubWorkflowGenerate") ++
         List("+headerCreateAll").filter(_ => header) ++
         List("+scalafixAll").filter(_ => scalafix) ++
         List("+scalafmtAll", "scalafmtSbt").filter(_ => scalafmt) ++
-        List("javafmtAll").filter(_ => javafmt)
+        List("javafmtAll").filter(_ => javafmt) ++
+        List("compile", "test") ++
+        List("mimaReportBinaryIssues").filter(_ => mima)
+    },
+    GlobalScope / tlCommandAliases ++= {
+      val header = tlCiHeaderCheck.value
+      val scalafmt = tlCiScalafmtCheck.value
+      val javafmt = tlCiJavafmtCheck.value
 
       val botHook = List("githubWorkflowGenerate") ++
         List("+headerCreateAll").filter(_ => header) ++
+        List("javafmtAll").filter(_ => javafmt) ++
         List("+scalafmtAll", "scalafmtSbt").filter(_ => scalafmt)
```

Comment:
Steward only runs the bot hook only when there is an sbt-typelevel update PR. sbt-typelevel includes `sbt-scalafmt` but it doesn't include sbt-java-formatter. So there isn't really a situation where `javafmtAll` would need to be run to commit changes after an update. On the other hand, it's probably a harmless sink of Steward CPU cycles, esp. since few projects are using javafmt anyway.

Reply:
Happy to remove it, just felt weird without the symmetry....

Reply:
Yeah, they are not really meant to be symmetrical. `prePR` is intended for human-made changes, `tlPrePrBotHook` is intended for bot-made changes. Anyway, I think it's fine to leave.